src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayprice = 40;
+
+  if (days >= 3 && days < 7) {
+    return dayprice * days - 20;
+  }
+
+  if (days >= 7) {
+    return dayprice * days - 50;
+  }
+
+  return dayprice * days;
```

Comment:
nice) 
 Yet it would be even better if you create a variable for repeating part of code `dayprice * days` and for `50` `20`
 ```
 const initialPrice = dayprice * days
 const WEEK_RENTAL = 50
 const DAY_ RENTAL = 20
 ```
 I wrote in uppercase since they are constants (non-changed values)